### PR TITLE
Adding support for openshift version

### PR DESCRIPTION
**Pull request by tsorya**

### CODE REVIEW

File: src/installer/installer.go [hunk 1]
```diff
@@ -83,9 +83,15 @@ func (i *installer) InstallNode() error {
 	if err != nil {
 		return err
 	}
+
 	i.UpdateHostStatus(WritingImageToDisk)
+	image, err := i.getRchosImage()
```
Comment (eranco74) [inline src/installer/installer.go:88]: Please move to another package.
The mapping and the helper functions.
you can name it "supportedOpenshiftVersion"

File: src/installer/installer.go [hunk 2]
```diff
@@ -97,6 +103,15 @@ func (i *installer) InstallNode() error {
 	return nil
 }
 
+func (i *installer) getRchosImage() (string, error) {
```
Comment (eranco74) [inline src/installer/installer.go:106]: Let's have a similar function for the MCO.
I think the map should look like this:
```
{4.4: {"rhcos": "https://blabla",
        "mco": "mco-image"},
4.5: ...
}
```

File: src/installer/installer_test.go [hunk 3]
```diff
@@ -24,15 +25,17 @@ func TestValidator(t *testing.T) {
...
+		ctrl             *gomock.Controller
+		mockops          *ops.MockOps
+		mockbmclient     *inventory_client.MockInventoryClient
+		mockk8sclien     *k8s_client.MockK8SClient
+		i                *installer
+		bootstrapIgn     = "bootstrap.ign"
+		masterIgn        = "master.ign"
+		workerIgn        = "worker.ign"
+		openShiftVersion = "4.4"
+		rhcos, _         = utils.GetRhcosImageByOpenshiftVersion(openShiftVersion)
```
Comment (eranco74) [inline src/installer/installer_test.go:38]:
```suggestion
		image, _         = utils.GetRhcosImageByOpenshiftVersion(openShiftVersion)
```

File: src/utils/openshiftVersion.go [hunk 4]
```diff
@@ -0,0 +1,35 @@
+package utils
```
Comment (eranco74) [inline src/utils/openshiftVersion.go:1]: Rename the file, no need for camalCase.

File: src/installer/installer_test.go (hunk 3 above)
Comment (tsorya) [reply src/installer/installer_test.go:38]: :) it was image and i changed it to rhcos

File: src/utils/openshiftVersion.go (hunk 4 above)
Comment (tsorya) [reply src/utils/openshiftVersion.go:1]: ok

**Pull request merged**